{
  "metadata": {
    "timestamp": "2025-02-25T04:14:31Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2020-25707",
        "description": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. Reason: This candidate is a duplicate of CVE-2020-2891",
        "metadata": {
          "published_date": "2020-12-14T21:15:20.270",
          "last_modified_date": "2023-11-07T03:20:23.500",
          "confidence_level": "low",
          "severity": "n/a"
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": false
        },
        "counts": {
          "public_exploit_count": 0
        },
        "epss": {
          "score": 0.00241,
          "percentile": 0.63757
        },
        "metrics": [],
        "timeline": {
          "references": [],
          "repositories": []
        }
      }
    }
  ]
}